InAppUtils/InAppUtils.m

```diff
@@ -233,7 +233,7 @@ - (void)productsRequest:(SKProductsRequest *)request
                                       @"currencySymbol": [item.priceLocale objectForKey:NSLocaleCurrencySymbol],
                                       @"currencyCode": [item.priceLocale objectForKey:NSLocaleCurrencyCode],
                                       @"priceString": item.priceString,
-                                      @"priceLocale": item.priceLocale.countryCode,
+                                      @"countryCode": item.priceLocale.countryCode,
```

Comment:
1) consistency with the rest of the codebase.
2) if they decide to change the key name, using an identifier(NSLocaleCountryCode) ensures we are not breaking our code?